### PR TITLE
Deep jani05 patch 1

### PR DESCRIPTION
**Pull request by DeepJani05**
I hope I gave it my best

### CODE REVIEW

File: main .md [hunk 1]
```diff
@@ -0,0 +1,396 @@
...
+scor=reqr.score(x_test,y_test)
+print(f"\t linear regression prediction\n",scor)
+
+
+#lassso re
+LS=Lasso(alpha=0.05)
+LS.fit(x_train,y_train)
+v=LS.score(x_test,y_test)
+print(v,f"\t lassso regression prediction\n")
+ #gradientBoostingRegressor
```
Comment (DeepJani05) [inline main .md:366]: #gradientBoostingRegressor

from xgboost import XGBClassifier
seed=7
text_size=0.33
x1 = train.iloc[:,0:8]
y2 = train.iloc[:,8]
print(x1)
print(y2)

x_train,x_test,y_train,y_test=train_test_split(x1,y2,test_size=text_size,random_state=seed)

model=XGBClassifier()
model.fit(x_train,y_train)
print(model)

y_predd=model.predict(x_test)
predicton=[round(value) for value in y_predd]
print(predicton)

accary=accuracy_score(y_test, predicton)
print("Accuracy: %.2f%%" % (accary * 100.0))

Comment (DeepJani05) [reply main .md:366]: using it in place of previous code
